tests/lua-tests/src/controller.lua

```diff
@@ -1,4 +1,7 @@
 
+-- jit off
+local jit = require("jit")
+jit.off()
```

Comment:
Why have to disable jit?

Reply:
JIT Capability Shutdown
Reason: To prevent malware from bypassing app market reviews by abusing JIT (RWX) memory to execute arbitrary code

Reply:
But why only this test case need to disable jit?

Reply:
disable jit  here will apply to all test cases.

Reply:
I think it is better:
- disable it in mainmenu.lua is better
- only disable it on OHOS

Reply:
done

Reply:
And i have question. If not support jit, why use luajit on OHOS, just use lua vm is enough.

Reply:
How can I only use the lua vm?

Reply:
Your meaning is that in CMake, we should not link LuaJIT? Will this have other impacts? LuaJIT has also made optimizations in other aspects.

Reply:
Yep, like iOS, it uses lua instead of luajit.

Reply:
It's also ok to use it as current implementation. Please make a decision and modify all PR.

Reply:
Also need use luajit. After disable jit, luaJIT remains more optimized. please see this bloc https://blog.csdn.net/u014397684/article/details/125686988